dev-docs/pb9-notes.md

```diff
@@ -0,0 +1,124 @@
+title: Prebid.js 9.0 Release Notes & Publisher API Changes
+description: Description of the breaking changes included for Prebid.js 9.0
+sidebarType: 1
+---
+
+# Prebid.js 9.0 Bidder Interface and Publisher API Changes
+
+{:.no_toc}
+
+This document describes the changes included for Prebid.js version 9.0.
+
+* TOC
+{:toc}
+
+## Removed Modules
+
+The following modules have been removed from Prebid.js as part of the 9.0 release. Publishers building with one of them will need to point to its replacement or remove the module from their build.
+
+{: .table .table-bordered .table-striped }
+| Module      | Replacement |
+|:-----------------------------------------|:---------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------|
+| Brightcom | OMS Bid Adapter |
+| Sovrn Analytics | |
+| Empty FPD module file | |
+| Adomik | |
+| Britepool | |
+| SpotX | |
+| pirId | |
+| Engage BDR | |
+| MyTarget | |
+| Parrable | |
+| Blue BillyWig | |
+| Idward RTD | Anonymized RTD |
+| Minutemedia Plus | |
+| eplanning Analytics | |
+| mars media Analytics | |
+| sigmoid Analytics | |
+| sonobi Analytics | |
+| staq Analytics | |
+| RichAudience | (Temporary) |
+| adbookpsp | |
+| yahooSSP | yahooAds |
+| GDPR Consent module | TCF Consent module |
+| GDPR Enforcement module | TCF Control module |
+| Bizzclick | Blasto |
+| Utiq | UtiqId |
+| Prebid Manager | AsterioBid Prebid Manager |
+| Ras | RingierAxelSpringer |
+| Fledge modules | PAAPI modules |
+| DFP Video | Split into DFP Video and DFP AdPod |
+
+## Consent changes:
+
+The USP string was removed from the consent metadata; also USP module is no longer in the recommended build. The GDPR modules were renamed to TCF modules, to reflect their adherence to a technical specification and not imply adherence to the underlying legislation and case law. Support for GPP 1.0 was removed from the GPP module. Of particular importance, "vendorless" modules such as the sharedid module no longer rely on vendor consent in the TCF object, but instead rely on publisher purpose consent. Publishers should check their __tcfapi consent data object to confirm publisher purpose consents are requested by their CMP.
+
+## Rendering:
+
+This (creative)[https://github.com/prebid/Prebid.js/blob/master/integrationExamples/gpt/creative_rendering.html] is now the preferred creative choice for Google Ad Manager users for web. It is created as part of the build process. This (legacy implementation)[https://github.com/prebid/Prebid.js/blob/8.52.0/integrationExamples/gpt/creative_rendering.html] was deleted and examples using the Prebid Universal Creative (PUC) in documentation are being ported over to the new guidance. Publishers also using Prebid Mobile SDK creative for apps may wish to remain on the PUC for ease of operations.
```

Comment:
> This (legacy implementation)[https://github.com/prebid/Prebid.js/blob/8.52.0/integrationExamples/gpt/creative_rendering.html] was deleted

it was updated, not deleted. it also is not related to PUC. the idea behind it is documented here: https://docs.prebid.org/adops/js-dynamic-creative.html

Reply:
It was deleted 
![image](https://github.com/prebid/prebid.github.io/assets/1683175/23bc41db-df38-42c6-9f15-ad644b14c162)

Reply:
the only remaining creative is https://github.com/prebid/Prebid.js/blob/master/integrationExamples/gpt/x-domain/creative.html; there were two, now there is one